python/lib/rt_m1_client/client.py

```diff
@@ -557,7 +557,7 @@ async def activateConsumptionReportingConfiguration(self, provisioning_session_i
             ret: ConsumptionReportingConfigurationResponse = self.__tag_and_date(result)
             ret['ConsumptionReportingConfiguration'] = ConsumptionReportingConfiguration.fromJSON(result['body'])
             return ret
-        elif result['status_code'] == 204:
+        elif result['status_code'] in [201, 204]:
```

Comment:
Could do with a comment explaining the reason for this weirdness.

Reply:
Comment added